MVP/sample_job.yaml

```diff
@@ -0,0 +1,28 @@
+- job-template:
+    name: '{name}-{component}'
+    defaults: defaults-build
+    properties:
+        - ownership:
+            owner: lilu
+            co-owners:
+                - lilu
+                - lilu
+    triggers:
+        - ci-trigger:
+            jms-selector: "name = '{component}' AND CI_TYPE = 'brew-tag' AND (tag LIKE 'RHEL-6._-candidate' OR tag LIKE 'rhel-7._-candidate')"
+    builders:
+        - shell: |
+            set -x
+
+            # Here goes your shell commands for submitting beaker job here, and save the job ID into env var BKR_JOBID
+            # E.g. `bkr workflow-tomorrow --distro=$distro --task-file=$taskfile $other_args`
+            BKR_JOBID="J:1000000"
+
+            bkr job-watch $BKR_JOBID
+            bkr job-results --format junit-xml $BKR_JOBID > $WORKSPACE/$BKR_JOBID.xml
```

Comment:
As I mentioned on the call, I really believe this is not a good idea. I understood the intent that we will keep the template unified and expose some fields to parametrize. Here we expose the whole template for user to modify anyhow she likes, defeating the whole purpose of it being a template - there are no shared elements and we may end (again) with the job being entirely different.  There is no parametrization.

My understanding was we will keep the template central and let the user to modify only the parameters in the 'project' section.